core/src/main/java/com/linecorp/armeria/server/metric/ThreadDumpService.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License
+ */
+
+package com.linecorp.armeria.server.metric;
```

Comment:
I'm not sure `metric` is a good package for `ThreadDumpService`.
Do we need a new package such as `monitor`?

Reply:
How about [`management`](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/management/ThreadDumpEndpoint.java)?

Reply:
`management` is a good candidate. Java also use `java.lang.management.ThreadInfo`.